images/update-drivers-website/index.html

```diff
@@ -44,7 +44,7 @@
         var url = new URL(window.location);
         var lib = url.searchParams.get('lib');
         if (lib == null) {
-            lib = "3.0.0+driver";
+            lib = "3.0.1+driver";
```

Comment:
Isn't there a way to compute latest release given a semver? I mean, can we automate this?

Reply:
Maybe by extracting the lib version used in last falco release, do you have the link where it's set for compilation?